src/cpp/server/server_cc.cc

```diff
@@ -473,7 +473,8 @@ std::shared_ptr<Channel> Server::InProcessChannel(
     const ChannelArguments& args) {
   grpc_channel_args channel_args = args.c_channel_args();
   return CreateChannelInternal(
-      "inproc", grpc_inproc_channel_create(server_, &channel_args, nullptr));
+      "inproc", grpc_inproc_channel_create(server_, &channel_args, nullptr),
+      nullptr);
```

Comment:
Again, wouldn't we also want an InProcess channel to allow interceptors?

Reply:
added